src/_guides/language/language-tour.md

```diff
@@ -3854,6 +3874,32 @@ main() {
 }
 {% endprettify %}
 
+### Typedefs and generic functions
+
+To further support first-class generic functions, Dart has added
+syntax, called _function-type syntax_, allowing you to refer
+to a function's types in a typedef statement.
+This syntax uses an assignment operator in conjunction with
+`Function()` in the declaration.
+
+Compare the following typedef statements:
+
+|--------------------------+------------------------------------------------|
+| Typedef example          | Description                                    |
+|--------------------------|------------------------------------------------|
+| <code>typedef Foo<T>(T param); | A function that takes `dynamic`. |
+| <nobr><code>typedef Foo = Function<T>(T param); | A generic function that types some type parameter T, and a parameter of that type. |
```

Comment:
The descriptions will need to be revisited. I have some comments and questions, but I'll post them to the original issue (https://github.com/dart-lang/site-www/issues/302).